book/src/SUMMARY.md

```diff
@@ -1,21 +1,36 @@
 # Kubernetes/CSI Book
 
-- [Introduction](Home.md)
-- [Using a CSI Driver](UsingCSIDriver.md)
+- [Introduction](introduction.md)
+- [Developing CSI Driver for Kubernetes](developing.md)
+    - [Sidecar Containers](sidecar-containers.md)
+        - [external-provisioner](external-provisioner.md)
```

Comment:
Is there any order for listing the sidecars?

Reply:
Grouped all the `external-*er`, followed by all the `*driver-registrar`, and then other.